### ISSUE TITLE
Solar API feed looks broken and showing 0.00

### ISSUE THREAD

**Issue by jv21** [gjlipman/smartmeter#3]

**Comment by jv21** [comment]
Forcast broken for solar based on UV value. I think this is a great site for predicitng what day is best to charge an EV

**Comment by t288msd** [comment]
Agree. with solar providing most of the dip in pricing at the moment (1p/kWh today) the forecast is quite out. I charged yesterday at 12p/kWh only to find its 1p/kWh today. I looked to fix and raise a pull request but my coding is very rusty so I couldn't find where the solar comes from. Fingers cross someone else can fix
